llama-index-integrations/vector_stores/llama-index-vector-stores-vespa/llama_index/vector_stores/vespa/templates.py

```diff
@@ -94,3 +94,87 @@
         )
     ],
 )
+
+# I am not sure what to name this, so I decided a name that highlights the difference with hybrid_template
+# This needs to be renamed to something more meaningful
+with_fields_template = ApplicationPackage(
```

Comment:
I think this is good for now, and we can improve template naming once we get a better feel for use cases.